chromadb/api/types.py

```diff
@@ -65,6 +65,10 @@ def maybe_cast_one_to_many_embedding(
         if isinstance(target[0], (int, float)):
             return cast(Embeddings, [target])
     # Already a sequence
+
+    if isinstance(target, np.ndarray):
```

Comment:
this logic is not right not got rectified in the following PR: https://github.com/chroma-core/chroma/blob/45f5d5551ba54bb5f61524b7bb5ccb4caff01200/chromadb/api/types.py#L56

Reply:
Can we move the correct logic into this PR so we don't merge broken code?

Reply:
done